src/scripts/main.js

```diff
@@ -1,3 +1,390 @@
 'use strict';
 
-// write your code here
+const start = document.querySelector('.start');
+const score = document.querySelector('.game-score');
+const messageStart = document.querySelector('.message-start');
+const messageLose = document.querySelector('.message-lose');
+const messageWin = document.querySelector('.message-win');
+
+score.textContent = 0;
+
+const cells = {
+  1: '',
+  2: '',
+  3: '',
+  4: '',
+  5: '',
+  6: '',
+  7: '',
+  8: '',
+  9: '',
+  10: '',
+  11: '',
+  12: '',
+  13: '',
+  14: '',
+  15: '',
+  16: '',
+};
+
+const colorMods = [
+  'field-cell--2',
+  'field-cell--4',
+  'field-cell--8',
+  'field-cell--16',
+  'field-cell--32',
+  'field-cell--64',
+  'field-cell--128',
+  'field-cell--256',
+  'field-cell--512',
+  'field-cell--1024',
+  'field-cell--2048',
+];
+
+function getRandom(min, max) {
+  return Math.round((min - 0.5) + Math.random() * (max - min + 1));
+}
+
+function removeColors() {
```

Comment:
you can query all cells, iterate through the list, and set the default class for each
and get rid of `colorMods ` array